### ISSUE TITLE
Bien ahi el personal trainer

### ISSUE THREAD

**Issue by rogarciaa** [obj1unq/2021s1-presentacion-lautarovillagram#2]
Laburas de personal trainer? Que es lo que mas te gusta de eso?

**Comment by lautarovillagram** [comment]
No, actualmente no estoy ejerciendo, en su momento cuando me recibí busque trabajo por todos lados pero difícil entrar a trabajar en un gimnasio si no conoces al dueño, despues como estaba lenta la busqueda me puse a estudiar esto, y despues paso la pandemia, asi que tengo el titulo de prof de musculacion + personal trainer pero nunca lo pude practicar bien jajaja. Lo que mas me gusta es el armado y diagramado de rutinas, depende el objetivo de la persona o si tiene alguna dificultad/limitacion
